include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -3651,13 +3651,19 @@ struct OrtApi {
    *     - "73"
    *     - "75"
    *   "device_id": The ID of the device to use when setting 'htp_arch'. Defaults to "0" (for single device).
-       "enable_htp_fp16_precision": Used for float32 model for HTP backend.
-       Enable the float32 model to be inferenced with fp16 precision. Otherwise, it will be fp32 precision.
-         - "0": With fp32 precision.
-         - "1": Default. With fp16 precision.
-       "enable_htp_weight_sharing": Enable QNN weight sharing feature while compiling multiple graphs into one QNN context.
-         - "0": Default. Disabled.
-         - "1": Enabled.
+   *   "enable_htp_fp16_precision": Used for float32 model for HTP backend.
+   *   Enable the float32 model to be inferenced with fp16 precision. Otherwise, it will be fp32 precision.
+   *     - "0": With fp32 precision.
+   *     - "1": Default. With fp16 precision.
+   *   "enable_htp_weight_sharing": Enable QNN weight sharing feature while compiling multiple graphs into one QNN context.
+   *     - "0": Default. Disabled.
+   *     - "1": Enabled.
+   *   "offload_graph_input_quantization": Offload graph input quantization to another execution provider (typically CPU EP).
+   *     - "0": Default. Disabled. QNN EP will handle quantization of graph inputs.
+   *     - "1": Enabled.
+   *   "offload_graph_output_dequantization": Offload graph output dequantization to another execution provider (typically CPU EP).
+   *     - "0": Default. Disabled. QNN EP will handle dequantization of graph outputs.
+   *     - "1": Enabled.
```

Comment:
unrelated, but since we remove SNPE, i think we should remove all the doc lines below related to SNPE